src/main/java/com/starkinfra/PixBalance.java

```diff
@@ -39,13 +39,14 @@ public final class PixBalance extends Resource {
      * workspace. The balance is never generated by the user, but it
      * can be retrieved to see the information available.
      * <p>
-     * Attributes (return-only):
+     * Parameters:
      * @param id [string]: unique id returned when PixBalance is created. ex: "5656565656565656"
-     * @param amount [long]: current balance amount of the workspace in cents. ex: 200 (= R$ 2.00)
+     * @param amount [Long]: current balance amount of the workspace in cents. ex: 200 (= R$ 2.00)
      * @param currency [string]: currency of the current workspace. Expect others to be added eventually. ex: "BRL"
      * @param updated [string]: update datetime for the balance. ex: "2020-03-10 10:30:00.000000+00:00"
      */
-    public PixBalance(long amount, String currency, String updated, String id) {
+    public PixBalance(Long amount, String currency, String updated, String id
+    ) {
         super(id);
         this.amount = amount;
```

Comment:
Como que fica o tratamento sendo que aqui o tipo é Long e não tem aquela inicialização assim:
`this.amount = ((Number) dataCopy.remove("amount")).longValue();`
? ele funciona normal?

Pra você entender, como aqui Não têm input do usuário, não precisa daquela terceira Docstring e função que faz essa inicialização. 
Esse formato pode ser aplicado para os outros recursos que não tem create (inout do usuário) também, como IssuingPurchase, CardMethod, MerchantCategory, MerchantCountry e IssuingBalance para simplificar

Reply:
Funciona normal. 

Dei um check nesses recursos e nenhum deles tem a 3ª docstring.